### DISCUSSION TITLE
past clipboard icon in top bar

### DISCUSSION THREAD

**Discussion by Staini** [AppGround-io/bluetooth-keyboard-and-mouse-support#208] [Ideas]
I like your past clipboard function very much. But I hate your text input bar.

In the first versions of your app you had a past clipboard icon in the top bar.
Can you please add an option to use the past clipboard icon in the top bar instead of the text input bar (disable the bar)?

And thank you for your app. I have been using it for years now.

**Comment by zorazorr** [comment]
A button to toggle the visibility of the bar would probably be more common.

Nice idea... I sometimes accidentally hit the "direct" button, which is confusing for the moment.

**Reply by Staini** [reply]
> I sometimes accidentally hit the "direct" button, which is confusing for the moment.

I have the same problem.

**Comment by Staini** [comment]
I just had an idea :-)
Can a long click on the keyboard icon toggle the textbar option complete off / on?
![20220222_115257](https://user-images.githubusercontent.com/91640388/155118245-314ac2da-8dd3-4793-a89a-6f49188005eb.jpg)

This would eliminate the need to always go to options, to change the setting depending on the purpose of use.
